maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -354,12 +354,16 @@ data class PressKeyCommand(
 
 }
 
-data class EraseTextCommand(
-    val charactersToErase: Int,
+data class EraseTextCommand (
+    val charactersToErase: Int?,
 ) : Command {
 
     override fun description(): String {
-        return "Erase $charactersToErase characters"
+        return if (charactersToErase != null) {
+            "Erase $charactersToErase characters"
+        } else {
+            "Erase 50 characters by default"
```

Comment:
Let's say `Erase text` if `charactersToErase` is null

Reply:
Done ✅